CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Welcome to CONTRAST RATIO APP's contributing guide
+
+:+1::tada: First off, thank you for investing your time in contributing to our project! :tada::+1:
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+Use the table of contents icon on the top left corner of this document to get to a specific section of this guide quickly.
+
+## Code of Conduct
+
+Please, read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+## New contributor guide
+
+To get an overview of the project, please read the [README](./README.md).
+
+## Getting started
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with the docs, search if an issue already exists. If a related issue doesn't exist, you can open a [new issue](https://github.com/DBenMoshe/contrast-ratio-repo/issues/new).
```

Comment:
Why are we talking only about the problem with the docs here? Shouldn't we talk about problem with our codebase in general? Or it's just my english going off?

Reply:
I also wasn't sure if the docs was used only by Github to refer to the Github docs or not.
So, if it is better to use the codebase word instead I can change it, just let me know.